.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,64 @@
+name: "Feature Request"
+description: "Have a great new idea? We'd love to know, please search through our existing feature requests, and upvote it if its already been submitted."
+labels: ["-x-new-feature", "triage"]
+assignees: ["sylwiabr", "xvcgreg"]
+body:
+  - type: "input"
+    id: "discord-id"
+    attributes:
+      label: "Discord username (optional)"
+      description: "If you're in our [Discord server](https://discord.gg/enso), please include your [Discord ID](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) formatted like so: `username#1234` (you can find it by clicking your profile on the bottom left of the application). This is so we can ask questions interactively and give you the `Ideas Contributor` Discord role."
+      placeholder: "username#1234"
+    validations:
+      required: false
+
+  - type: "dropdown"
```

Comment:
I don't think this is particularly useful and is very subjective. At the end of the day prioritization will be done on our side based on the provided description and understanding of the feature request.

Reply:
Sure, but it shows how much this thing is important subjectively to someone asking. I would still leave it for now. It is working good on ther repos, e.g. warp temirnal - and I really like how people are using it.